### DISCUSSION TITLE
Missing russian translation?

### DISCUSSION THREAD

**Discussion by 0x4E69676874466F78** [Blinue/Magpie#260] [Dev Channel]
https://github.com/Blinue/Magpie/blob/bf3aa903a64ad77f14f1f0154aeb28b49a5e8597/Magpie/Properties/Resources.ru-RU.resx
https://github.com/Blinue/Magpie/tree/main/Magpie/Properties ?

**Comment by Blinue** [comment]
I'v removed this local since no one is maintaining it.

**Reply by 0x4E69676874466F78** [reply]
Is the translation really outdated?

**Reply by Blinue** [reply]
#68 
The Russian translation has not been updated since it was added.

**Reply by 0x4E69676874466F78** [reply]
Okay, I maked new translation #263
